sass-loader module not found with v2.0.0-beta.4

## Describe the bug

'sass-loader' module required by webpack config is not found 

Offending line seems to be: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/blob/4ada6337d286941f37efd26c502274d05c30375a/packages/docusaurus-theme-openapi-docs/src/index.ts#L53

## Expected behavior

No exception.

## Current behavior

```
[ERROR] Error: Cannot find module 'sass-loader'
Require stack:
- /node_modules/.pnpm/docusaurus-theme-openapi-docs@2.0.0-beta.3_@docusaurus+core@2.4.0_@types+react@18.2.22_eslint_s5oi2hxuky52ztbp5hpktak3sa/node_modules/docusaurus-theme-openapi-docs/lib/index.js
- /node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/server/plugins/configs.js
- /node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/server/plugins/init.js
- /node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/server/plugins/index.js
- /node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/server/index.js
- /node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/commands/build.js
- /node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Function.resolve (node:internal/modules/cjs/helpers:127:19)
    at Object.configureWebpack (/node_modules/.pnpm/docusaurus-theme-openapi-docs@2.0.0-beta.3_@docusaurus+core@2.4.0_@types+react@18.2.22_eslint_s5oi2hxuky52ztbp5hpktak3sa/node_modules/docusaurus-theme-openapi-docs/lib/index.js:47:57)
    at applyConfigureWebpack (/node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/webpack/utils.js:133:43)
    at /node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/commands/build.js:155:62
    at Array.forEach (<anonymous>)
    at buildLocale (/node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/commands/build.js:149:13)
    at async tryToBuildLocale (/node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/commands/build.js:41:20)
    at async mapAsyncSequential (/node_modules/.pnpm/@docusaurus+utils@2.4.0_@docusaurus+types@2.4.0/node_modules/@docusaurus/utils/lib/jsUtils.js:34:24)
    at async Command.build (/node_modules/.pnpm/@docusaurus+core@2.4.0_@docusaurus+types@2.4.0_eslint@8.48.0_react-dom@17.0.2_react@17.0.2_typescript@4.9.5/node_modules/@docusaurus/core/lib/commands/build.js:76:21)
```

## Possible solution

Sounds related to the use of `pnpm` as the demo is working with `yarn`.

## Steps to reproduce

<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->

1. Run a new project with `pnpm`

## Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

- Version used: `v2.0.0-beta.4`
- Environment name and version: pnpm8 node18

--- a/packages/docusaurus-theme-openapi-docs/package.json
+++ b/packages/docusaurus-theme-openapi-docs/package.json
@@ -63,6 +63,7 @@
     "redux-devtools-extension": "^2.13.8",
     "rehype-raw": "^6.1.1",
     "sass": "^1.58.1",
+    "sass-loader": "^13.3.2",
     "webpack": "^5.61.0",
     "xml-formatter": "^2.6.1"
   },

